exercises/concept/interest-is-interesting/interest_is_interesting_test.go

```diff
@@ -8,7 +8,8 @@ import (
 const floatEqualityThreshold = 1e-5
 
 func floatingPointEquals(got, want float64) bool {
-	return math.Abs(got-want) <= floatEqualityThreshold
+	return math.Abs(got-want) <= floatEqualityThreshold ||
+		math.Abs(got-want)/(math.Abs(got)+math.Abs(want)) <= floatEqualityThreshold
```

Comment:
I would recommend to improve readability by creating two variables, "absoluteDifferenceBelowTreshold" and "relativeDifferenceBelowTreshold" or similar and then using those in the actual return statement with the `||` operator.

Reply:
Sent the version with two variables :) Thank you, mentoring feedback is exactly what I'm looking for.